src/apps/default.erl

```diff
@@ -38,7 +38,15 @@ init([]) ->
   ?D(<<"Starting default application">>),
   {ok, #state{}}.
 
-authorize(Client,_) -> gen_server:call(?SERVER, {authorize, Client}).
+%% authorize(Client,_) -> gen_server:call(?SERVER, {authorize, Client}).
+
+authorize(Client, Data) ->
+  AuthModules = ?Config(default_app, []),
+  Auth = not lists:any(fun({Module, Args}) -> not Module:verify(Client, Data, Args) end, AuthModules),
```

Comment:
Line 45 is too long:   Auth = not lists:any(fun({Module, Args}) -> not Module:verify(Client, Data, Args) end, AuthModules),.